src/common/components/nav/nav.component.js

```diff
@@ -174,20 +174,11 @@ class NavController{
     }))
       .map((response) => {
         let jsonStructure = response.data;
-        let menuStructure = {
-          main: jsonStructure['/content/cru/us/en'],
+        return {
+          main: jsonStructure['/content/cru/us/en'] || jsonStructure['main'],
           global: jsonStructure['/content/cru/us/en/global'],
           give: jsonStructure['/content/give/us/en']
         };
-
-        //add give to main nav
-        menuStructure.main.push({
-          title: 'Give',
-          path: '/give',
-          children: jsonStructure['/content/give/us/en']
-        });
```

Comment:
Ok, got it, but if this section code is removed, we must change the endpoint at cru.org and give sites to add give menu option to the menu. I'd like to leave that navigation generation endpoint at AEM side as it currently is to avoid risks.